### ISSUE TITLE
Detect via JS native in browser

### ISSUE THREAD

**Issue by eduardo-mior** [sindresorhus/file-type#478]
I have an application using native Javascript.

How could I use this Client-side package to detect the MimeType of an uploaded file?

I only found examples using Node, is there any way to do this using native Javascript?

**Comment by sindresorhus** [comment]
There are browser examples in the readme too.

**Comment by eduardo-mior** [comment]
@sindresorhus But I'm not using NPM.


There is no link to any CDN like all other JS libraries?

**Comment by sindresorhus** [comment]
There's no standalone browser build. You need a bundler like Webpack or Parcel.

**Comment by eduardo-mior** [comment]
Why don't you provide a Browser Build?
  
  
Every library I've worked with provides an independent distribution.

**Comment by Borewit** [comment]
> Every library I've worked with provides an independent distribution.

Sooner or later, you will find out that these standalone distributions are not a scalable solution. For a few simple HTML tricks this maybe sufficient, building web applications, this is an absolute no go.
Better to invest your time learning to work with a module bundlers (like Webpack or Parcel) @eduardo-mior.

**Comment by eduardo-mior** [comment]
Yes I understand this is not scalable.
But not everyone needs to make a giant system. There are many people (like me) who just need to make a simple system that is no more than 2 or 3 files.
  
  
You can't force everyone to learn to use Webpack or Parcel or other technologies, that's unfair.
  
  
As I explained earlier, almost all libraries provide a standalone distribution. I don't understand why you couldn't do that and collaborate on small projects.

**Comment by Borewit** [comment]
Please read: [Monolithic vs Modular]( https://medium.com/@berto168/monolithic-vs-modular-9b6d69684a2c) @eduardo-mior.

We provide module which is primarily designed to run in Node.js. 
If you believe you do the world a favor by creating a standalone distributions based on this module, just got for it.

I think it would not add any value, it will never be the right composition, and pointing folks like yourself in the wrong direction.

**Comment by eduardo-mior** [comment]
You don't need a battle tank to kill an ant. Many projects are simple and do not exceed 1 or 2 pages, or 1 or 2 html and js files.


Not all systems and sites need to be modular. This is a big problem for today's developers, they are forcing everyone to use React and Angular and other massive and complex technologies when what is needed is just an html file written hello world.
  
  
You already have all the knowledge of how Webpack and Parcel work, for you it's much easier to simply generate a standalone distribution. I don't have time at the moment to study how this all works. You could do this favor and generate the distribution, but since you don't want to do that, I'll copy the MimeType detection logic and put it in a JS file of my project and that's it.

**Comment by eduardo-mior** [comment]
I've worked with monolithic and modular databases and backend. Many companies take more than 1 year to develop a modular system that is retired soon after, while some companies take less than 2 months to make a monolithic system that already returns results.
 
 
I've already mentioned this in some lectures and meetings, we don't need to kill an ant with a cannon, not everything needs to be modular.
  
  
If you want to generate a standalone distribution I believe more people would use the library. At the moment I made my JS file based on your algorithm and I am abandoning this discussion.
